src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const greatDiscount = 50;
+  const rentalCost = 40;
+  const total = rentalCost * days;
+
+  if (days >= 7) {
+    return total - greatDiscount;
+  } else if (days >= 3) {
+    return total - basicDiscount;
+  }
+
+  return total;
```

Comment:
Instead of using 'else if' on line 12, consider returning from the function immediately after each condition. This reduces the level of indentation and makes the code cleaner and easier to read.
```suggestion
  if (days >= 7) {
    return total - greatDiscount;
  } 
  
  if (days >= 3) {
    return total - basicDiscount;
  }

  return total;
```